backend/src/routes/user.ts

```diff
@@ -21,17 +23,48 @@ router.get('/:username', async (ctx, next) => {
 	ctx.body = user;
 });
 
-router.put('/:username', async (ctx, next) => {
+//Create a new user with required handle and display name
+router.put('/newuser/:handle/:displayname', async (ctx, next) => {
+	console.log('new user route');
 	const user = new User();
-	user.username = ctx.params.username;
-	user.name = 'John Doe';
+	user.handle = ctx.params.handle;
+	user.displayname = ctx.params.displayname;
 	await user.save();
 	ctx.body = user;
 });
 
+//Change displayname by handle
+router.put('/change/:handle/:displayname', async (ctx, next) => {
```

Comment:
this is actually redundant with the "newuser" route, so i think we can just have one